### PR TITLE
Peanuts and Rocknuts

### PR DESCRIPTION
**Pull request by RealestEstate**
Adds peanuts to the seedvendor. They can also then be mutated into rocknuts, which contain iron.
also gives the weed plant its own sprite instead of placeholder.
credit to ComradeDispenser for the icons.

[sprites]
[content]

Edit:
-Eating rocknuts now does some damage and has a chance to break your teeth, higher chance at higher potencies.
-throwforce now scales with potency for rocknuts.
-added a new sound for cracking teeth, and hearing it makes my skin crawl every time.

:cl:
 * rscadd: After a 200 year ban of peanuts, NT has re-introduced them into stations to test their supposed allergical effects in a live study.
 * rscadd: Xenobotanists have recently discovered a mutant species of peanuts that may have actually existed millions of years ago. This new subspecies is named the rocknut because of its rock hard exterior. We suggest that you do not eat the rocknuts raw if you cannot afford dental coverage.

### CODE REVIEW

**Comment by SGuivenne** [comment]
Does eating a rock nut break your teeth/Cause you to lose them?

**Comment by RealestEstate** [comment]
no it’ll just warn you. but that’s an awfully devilish idea

**Comment by Goldsteel** [comment]
Eating rocknuts on help intent should warn you, and if it's harm intent it breaks a random amount of teeth and do damage.

**Comment by GermanSanta** [comment]
7kdXzK9

File: code/modules/hydroponics/seeds.dm [hunk 1]
```diff
@@ -1089,6 +1094,23 @@
 	potency = 10
 	ideal_light = 8
 
+
+/datum/seed/rocknut
+	name = "rocknut"
+	seed_name = "rocknut"
+	display_name = "quarry bush"
+	plant_dmi = 'icons/obj/hydroponics/rocknut.dmi'
+	products = list(/obj/item/weapon/reagent_containers/food/snacks/grown/rocknut)//TODO
```
Comment (Zth--) [inline code/modules/hydroponics/seeds.dm]: TODO what?

**Comment by Zth--** [comment]
I'd really like to suggest using the potency of the rocknut for something, like for example improving `throwforce`

File: code/modules/hydroponics/seeds.dm (hunk 1 above)
Comment (RealestEstate) [reply code/modules/hydroponics/seeds.dm]: to do delete this comment

**Comment by Blithering** [comment]
Popcorn has mild brute damage on the offchance you bite into an unpopped kernel, maybe rock nuts should have similar?

**Comment by Kurfursten** [comment]
let us know if you ever finish this

**Comment by RealestEstate** [comment]
yeah, just getting a bit carried away with finals and procrastination

**Pull request merged**